terraform/aws/modules/atlantis/atlantis.tf

```diff
@@ -69,3 +69,127 @@ resource "helm_release" "atlantis" {
   timeout = 1200
 
 }
+
+resource "aws_instance" "atlantis" {
```

Comment:
Why do we need this if we are using ASGs?

Reply:
To avoid deletion of existing instance.

Reply:
> To avoid deletion of existing instance.

But we can just create the ASG, etc and register the existing instance with the ASG. Because how this code is going to work in the future?